routing_common/transit_types.hpp

```diff
@@ -255,40 +255,89 @@ class ShapeId
   StopId m_stop2Id = kInvalidStopId;
 };
 
+class EdgeFlags
+{
+public:
+  DECLARE_TRANSIT_TYPE_FRIENDS
+
+  bool IsTransfer() const { return IsBit(kTransferMask); }
```

Comment:
Why not to use struct + bit-fields? It allows you to get rid of setters/getters + the logic will be simpler.

Reply:
Yes. Let's do so.

Reply:
Done. PTAL commit: "Using struct bit field for transit edge flags."

Reply:
But I we did not get rid of getter/setters. Do you propose to make the field public?
On the other hand methods for coding to uint8_t and decoding from uint8_t (GetFlags()/SetFlags()) became quite complicated. Do you have any idea how to make them simpler?